src/common/image.c

```diff
@@ -999,10 +999,10 @@ GList* dt_image_find_duplicates(const char* filename)
 #endif
 }
 
-static int _image_read_duplicates(const uint32_t id, const char *filename)
+// Search for duplicate's sidecar files and import them if found and not in DB yet
+static void _image_read_duplicates(const uint32_t id, const char *filename)
 {
   int count_xmps_processed = 0;
-  // Search for duplicate's sidecar files and import them if found and not in DB yet
```

Comment:
Doesn't seems related to your PR ?

Reply:
I've tried to fix some re-import behavior because that's not completely consistent but I gave up.
So nothing related, just a misplaced comment.

Reply:
I don't understand, now there is just comment removed, but I can swear that when I commented there was other lines removed that were part of e26bb3ae430 !!!! A GitHub bug ????